### DISCUSSION TITLE
What's the use-case for Blindspot?

### DISCUSSION THREAD

**Discussion by bradkoerner** [basbruss/adaptive-cover#252] [Q&A]
Based on a separate discussion, it seems that Minimal and Maximum Elevation are for factoring in things like a neighboring building or a fixed awning blocking the sun at a certain vertical angle in the sky.

I'm assuming that Field of view Left and Field of view Right do similar but for objects blocking the sun at a lateral angle from the window. Although I may be wrong there.

So then how does Blindspot differ from those parameters?

**Comment by basbruss** [comment]
The blind spot is a designated area in the field of view where the adaptive function is unnecessary because an object, such as something blocking the sun in front of your window, may be present.

**Reply by bradkoerner** [reply]
How does that differ from Minimal and Maximum Elevation and Field of view Left and Field of view Right?

**Reply by basbruss** [reply]
The field of view (FOV) sets the outer limits of what can potentially be observed from the window, while blindspots are regions within these limits where observation is impaired.
